async-nats/src/options.rs

```diff
@@ -347,6 +349,23 @@ impl ConnectOptions {
         self
     }
 
+    /// Sets the capacity for `Subscriptions`. Exceeding it will trigger `slow consumer` error
```

Comment:
`Subscriptions` are the internal write half of it so lets not talk about private types as code markup 🙂 

```suggestion
    /// Sets the capacity for `Subscribers`. Exceeding it will trigger `slow consumer` error
```